resources/views/course.blade.php

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+
+    <meta name="viewport" content="width=device-width, initial-
+scale=1, shrink-to-fit=no">
+
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="robots" content="noindex, nofollow">
+    <meta name="robots" content="noarchive">
+    <title>Profile</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/
+bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-
+JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css" rel=“stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/
+jquery.min.js"></script>
+</head>
+
+<body>
+    <div class="container">
+        <div class="card">
+            <h5 class="card-header">{{$courseAddr}} </h5>
+            <div class="card-body">
+                <p>Description: {{$courseDescription}}</p>
+                <p>Study Experience: {{$courseExperience}}</p>
+                <?php 
+                $output1 = '<p>Who took this: </p>';
+                if(is_string($courseMember)) {
+                    $output1 = '<p>Who took this: ';
+                    $output1 .= $courseMember . '</p>';
+                    echo $output1;
+                } else {
+                    foreach($courseMember as $a) {
+                        $output1 .= $a['first_name']. ' ' . $a['last_name'] . '<br/>';
+                    }
+                    echo $output1;
+                }
+                ?>
```

Comment:
以這種寫法的話，建議自己補上 `htmlspecialchars` 防止 XSS 
` $output1 .= htmlspecialchars($a['first_name']. ' ' . $a['last_name']) . '<br/>';`

當然也可以透過 blade 進行處理 {{ }}
```suggestion
               <p>Who took this: </p>
                <?php
                if(! is_string($courseMember) && count($courseMember)) {
                    foreach($courseMember as $student) {
                        ?>
                        {{ $student['first_name'] . ' ' . $student['last_name'] }} <br/>
                        <?php
                    }
                } else {
                    echo (string) $courseMember;
                }
                ?>
```